src/encoded/tests/test_submit.py

```diff
@@ -20,6 +20,10 @@
     xls_to_json
 )
 
+pytestmark = [pytest.mark.working]
+
+
+TEST_INGESTION_ID = '123456-1243-1234-123456abcdef'
```

Comment:
```suggestion
TEST_INGESTION_ID = '123456-1243-1234-123456abcdef'
TEST_INGESTION_ID2 = 'abcdef-1234-1234-abcdef123456'
```

Really I would call the first one `TEST_INGESTION_ID1` and then consistently rename it throughout the file, but that would require me to make too many suggestions here on this PR.  But I think you should definitely centralize the definition of these two quantities.